code/modules/clothing/masks/gasmask.dm

```diff
@@ -29,6 +29,14 @@
 	inhand_icon_state = "gas_cap"
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 
+/obj/item/clothing/mask/gas/driscoll
+	name = "driscoll mask"
+	desc = "Great for train hijackings. Works like a normal full face gas mask, but won't conceal your identity."
+	icon_state = "driscoll_mask"
+	flags_inv = HIDEFACIALHAIR
+	w_class = WEIGHT_CLASS_NORMAL

```

Comment:
```suggestion

```
this already inherits its w_class value from normal gas masks, you don't need to define it again